### ISSUE TITLE
What is the suitable length or depth of the long reads?

### ISSUE THREAD

**Issue by Yujiaxin419** [nadegeguiglielmoni/GraphUnzip#3]
Hello,

Thanks for developing this software. I am really interesting in it. So I following your methods in your biorxiv article (https://doi.org/10.1101/2021.01.29.428779) to assembled the _Solanum tuberosum_ genome. But I am confuse about why only the reads longer than 25kb be retained when used GraphAligner? meanwhile, the _Adineta vaga_ assembly used all long reads.

Is there a standard with long reads depth or length in using this tool?  Looking for your reply. Thanks a lot.

Yu Jiaxin

**Comment by nadegeguiglielmoni** [comment]
Hello,
In the case of _Solanum tuberosum_, we used only the reads larger than 25 kb as the amount of Nanopore reads was high and resulted in very large files. So we only used the reads larger than 25 kb to have smaller files to handle. 
In theory, the higher the depth, the better. Longer reads are also better, as they can overlap several segments, while shorter reads that map only to one segment will not be informative.
